### PR TITLE
cmd ref: change "Commands Reference" for "Command Reference" (all singular)

### PR DESCRIPTION
**Pull request by jorgeorpinel**

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
Do we need a (temporary) redirect in the DNS or app for old `/doc/commands-reference/...` paths?

**Comment by shcheklein** [comment]
👍 @jorgeorpinel yes, let's add a redirect in server.js for convenience

**Comment by jorgeorpinel** [comment]
Redirect added @shcheklein. Try https://dvc-org-pr-661.herokuapp.com/doc/command-reference fore example. Added some comments in the latest commit, which adds the redirect. Please review.

File: server.js [hunk 1]
```diff
@@ -39,16 +41,22 @@ app.prepare().then(() => {
...
       res.writeHead(301, {
         Location:
           'https://s3-us-east-2.amazonaws.com/dvc-public/' +
           req.headers.host.split('.')[0] +
           pathname
       })
       res.end()
-
-      // Special URL path redirects below
+    } else if (/^\/doc\/commands-reference.*/.test(pathname)) {
```
Comment (shcheklein) [inline server.js]: it will match `/commands-referencesdsdsdsd/something`, let's make it something like

`/^\/doc\/commands-reference(\/.*/)?`

?

Comment (jorgeorpinel) [reply server.js]: Sure, just need to make that `/` optional for path `/doc/commands-reference`. Fixed in 1eaca99.

Comment (jorgeorpinel) [reply server.js]: Oh wait I see your regex also achieves that. Looks better let me try that...

Comment (jorgeorpinel) [reply server.js]: Yes. (Except your last `/` should be after `?` 😋  Updated and tried with paths `/doc/commands-reference`, `/doc/commands-reference/add`, and `/doc/commands-referenceadd`. see latest commit.

**Pull request merged**